<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1303</width>
    <height>1045</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Tumblr Downloader</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>1000</width>
     <height>600</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>800</width>
          <height>400</height>
         </size>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Tumblr Downloader&lt;/span&gt; is an application that authenticates to your tumblr account, and downloads all the liked pictures to your computer.&lt;/p&gt;&lt;p&gt;In order to do this you need to register an application on the official tumblr website using the following link: &lt;a href=&quot;https://www.tumblr.com/oauth/apps&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#2980b9;&quot;&gt;https://www.tumblr.com/oauth/apps&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;p&gt;You should set the following fields as follows:&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Application Website&lt;/span&gt;: &lt;span style=&quot; font-style:italic;&quot;&gt;http://localhost:8080/tumblr&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Default Callback URL&lt;/span&gt;: &lt;span style=&quot; font-style:italic;&quot;&gt;http://localhost:8080/tumblr/callback&lt;/span&gt;&lt;/p&gt;&lt;p&gt;Once you have done that, fill the configuration parameters accordingly below. Detailed information regarding &lt;span style=&quot; font-weight:600;&quot;&gt;Tumblr API&lt;/span&gt; is found here: &lt;a href=&quot;https://www.tumblr.com/docs/en/api/v2&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#2980b9;&quot;&gt;https://www.tumblr.com/docs/en/api/v2&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <property name="openExternalLinks">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>Noto Sans [GOOG]</family>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Configuration</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>Noto Sans [GOOG]</family>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Status Log</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="blogNameLabel">
            <property name="toolTip">
             <string>This is your tumblr blog's domain name.</string>
            </property>
            <property name="whatsThis">
             <string/>
            </property>
            <property name="text">
             <string>Blog Name</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="blogNameLineEdit">
            <property name="toolTip">
             <string>This is your tumblr blog's domain name.</string>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Normal</enum>
            </property>
            <property name="placeholderText">
             <string>my-user-name.tumblr.com</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="consumerKeyLabel">
            <property name="text">
             <string>Consumer Key</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="consumerKeyLineEdit"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="consumerSecretLabel">
            <property name="text">
             <string>Consumer Secret</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="consumerSecretLineEdit"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="targetLocationLabel">
            <property name="text">
             <string>Target Location</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="targetLocationLineEdit"/>
          </item>
          <item row="6" column="1">
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Download</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="startOffsetLineEdit"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="startOffsetLabel">
            <property name="text">
             <string>Start Offset</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QCheckBox" name="debugModeCheckBox"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="debugModeLabel">
            <property name="text">
             <string>Debug Mode</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextEdit" name="statusTextArea"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1303</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
